Dockerfile.moc-xdmod

```diff
@@ -97,6 +98,22 @@ RUN chmod 774 /usr/bin/xdmod-openstack-hypervisor \
     && find /etc/httpd -type d -exec chmod 764 {} + \
     && find /etc/httpd -type f -exec chmod 664 {} +
 
+# openshift client
+RUN wget https://downloads-openshift-console.apps.ocp-staging.massopen.cloud/amd64/linux/oc.tar --no-check-certificate \
+    && tar xvf oc.tar \
+    && cp oc /usr/bin/oc \
+    && chmod g+rwx /usr/bin/oc
+
+# openshift reporting
+COPY ./xdmod-openshift-reporting.sh /usr/bin/xdmod-openshift-reporting.sh
+RUN git clone https://github.com/OCP-on-NERC/xdmod-openshift-scripts \
+    && cd xdmod-openshift-scripts \
+    && pip install -r requirements.txt \
+    && chmod g+rwx /usr/bin/xdmod-openshift-reporting.sh
+
+# force downgrade of mysql-connector-python
+RUN pip3 install -I mysql-connector-python==8.0.28
```

Comment:
Just for reference, what is the reason for this?

Reply:
Ah, there's a bug in 8.0.30 that makes mariadb unable to recognize the utf8 charset.

Reply:
Do you have a link to the bug so that we can track when it's resolved and can remove the workaround?

Reply:
Added a link to the issue; there doesn't seem to be a direct bug report that I can find.